tests/test_issues.py

```diff
@@ -136,3 +136,11 @@ def test_issue66(self):
     def test_issue70(self):
         f = uproot.open("tests/samples/issue70.root")
         self.assertEqual(f.keys(), [])
+
+    def test_issue74(self):
+        f = uproot.open("tests/samples/issue74.root")['Events']
+        self.assertTrue(isinstance(y, uproot.rootio.TVector3) for x in f.array('bees.xyzPosition') for y in x)
```

Comment:
How is it that `'bees.xyzPosition'` is a list (`x`) of `TVector3` (`y`) and then in the next check, each has a `fX`, `fY`, `fZ`?

Reply:
Could you please elaborate a bit? In the fX, fY, fZ check, I am just iterating over the contents of bees.xyzPosition. And the tests seem to be working. Did you mean I am already specifying x here and then using the same x to iterate in the next check?

Reply:
If you rewrite the code you have, it becomes more clear were the confusion is coming from.
lines 142-146 are logically identical to
```python
for x in f.array('bees.xyzPosition'):
    for y in x:
        # y is a TVector3
        self.assertTrue(isinstance(y, uproot.rootio.TVector3)
   # further down
    self.assertEqual(x.fX, 1.0) # x is also TVector3?
```
```